crypto/threads_none.c

```diff
@@ -133,4 +133,8 @@ int openssl_init_fork_handlers(void)
     return 0;
 }
 
+int openssl_get_fork_id(void)
+{
+    return return 0;
```

Comment:
Should this also call getpid() when OPENSSL_SYS_UNIX is defined?

Reply:
Sounds like a good idea. :-) I will add it.

Reply:
Done, see c42563119ea2b774da45f742e71b928ddf185efa.